mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -388,6 +388,13 @@ def ROCDL_mfma_f32_32x32x16_bf8_bf8 : ROCDL_Mfma_IntrOp<"mfma.f32.32x32x16.bf8.b
 def ROCDL_mfma_f32_32x32x16_bf8_fp8 : ROCDL_Mfma_IntrOp<"mfma.f32.32x32x16.bf8.fp8">;
 def ROCDL_mfma_f32_32x32x16_fp8_bf8 : ROCDL_Mfma_IntrOp<"mfma.f32.32x32x16.fp8.bf8">;
 def ROCDL_mfma_f32_32x32x16_fp8_fp8 : ROCDL_Mfma_IntrOp<"mfma.f32.32x32x16.fp8.fp8">;
+// New in gfx950
```

Comment:
The above comment says the fp8 instructions are only supported on gfx940. Do we want to update that to "new in gfx940"?

Reply:
Part of what's  new is the instructions supporting OCP FP8 instead of the gfx940 format

Reply:
So yeah, comment needs updating